### PR TITLE
Draft for quorum meetups

### PR DESCRIPTION
**Pull request by mjang**
Just a draft, I figure we need some reference from https://www.writethedocs.org/meetups/ 

For some reason, I can't get my local build working, so I haven't verified much.

We can add qrum-usWest.yml and qrum-EMEA.yml now or later, whenever you're ready.

### CODE REVIEW

File: docs/_data/meetups/qrm-usEast.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+region: Quorum
+country: USA
+state: East Coast
```
Comment (barbaricyawps) [inline docs/_data/meetups/qrm-usEast.yaml:3]: Technically we also include Canada. :)

File: docs/_data/meetups/qrm-usEast.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+city:
+website: https://www.meetup.com/virtual-write-the-docs-east-coast-quorum/
+meetup: Write-The-Docs-East-Coast
+twitter:
+organizers:
+  - name: Alyssa Whipple Rock
+    link: 
+  - name: Rose
```
Comment (barbaricyawps) [inline docs/_data/meetups/qrm-usEast.yaml:11]: Is the intent here to link to my professional website? If so, my website is: https://alyssarock.pro/

**Review by barbaricyawps — APPROVED** [review]
I just left some minor comments that aren't blockers. I approve!

File: docs/_data/meetups/qrm-usEast.yaml (hunk 2 above)
Comment (rosewms) [reply docs/_data/meetups/qrm-usEast.yaml:11]: Twitter would work for me https://twitter.com/ZelWms

Alternatively, we could reference the other meetups organizers work with.

Comment (barbaricyawps) [reply docs/_data/meetups/qrm-usEast.yaml:11]: Sounds good. Here's the list of participating organizers and chapters:

The following U.S. East Coast and Central meetups are currently participating in Quorum:

Atlanta, GA - Alan Bowman
Austin, TX - Shrie Spangler
Boston, MA - Rose, are you the organizer here?
Detroit, MI/Windsor, CAN - Ryan Macklin and Ashley Newton
Durham, NC - Susan Tacker
Florida - Barry Byron
Ottawa/Montreal, CAN - Shay Perez
Philadelphia, PA - Leslie Salazar

The following U.S. West Coast and Mountain meetups are currently participating in Quorum:

Bay Area, CA - Liz Harris, Clarence Cromwell, Maggie Fero
Los Angeles, CA - Andrea Kao
Portland, OR - Mike Jang
Salt Lake City, UT - Alyssa Rock

If you want links to the chapters and each of these individuals on Meetup, I recommend using the links from this page that I discovered recently: https://www.writethedocs.org/meetups/index.html

Alternatively, you can pull a partial list from the Quorum README: https://github.com/write-the-docs-quorum/quorum-meetups

File: docs/_data/meetups/qrm-usEast.yaml (hunk 1 above)
Comment (mjang) [reply docs/_data/meetups/qrm-usEast.yaml:3]: If I understand this setup correctly, we have to "pick one".

File: docs/_data/meetups/qrm-usEast.yaml (hunk 2 above)
Comment (mjang) [reply docs/_data/meetups/qrm-usEast.yaml:11]: The listing for the Quorum meetups, once I figure out the build, will be on the same page as the other meetups. So I think, for now, I'll "just" include a list to the other meetups (Quorum members).

Comment (mjang) [reply docs/_data/meetups/qrm-usEast.yaml:11]: Here's a screenshot of my WIP build:

QuroumMeetupChange

Adding everyone's organizer name, in the current YAML format, requires some detailed knowledge of rST and Sphinx that I don't have. The patterns don't look hard, but I've already spent a bunch of time on this admitted hack.

@barbaricyawps wdyt? @plaindocs I'd appreciate your comments as well, given the hackiness of what I'm doing.

I recognize this leaves open some other needed changes, such as adding Raleigh-Durham to the Meetup list.

Comment (barbaricyawps) [reply docs/_data/meetups/qrm-usEast.yaml:11]: I'm totally fine with leaving names off for now if it's easier. Thanks for doing this, @mjang !

Comment (barbaricyawps) [reply docs/_data/meetups/qrm-usEast.yaml:11]: FYI, I noticed most of the current organizers are listed here if you just want to link to this page: https://www.writethedocs.org/meetups/index.html

Comment (mjang) [reply docs/_data/meetups/qrm-usEast.yaml:11]: `Hi @barbaricyawps , this PR adds info to the bottom of that page.`

**Comment by mjang** [comment]
Hi @plaindocs If you're "good" with the hacks I've proposed in this PR, you're welcome to merge. If you have suggestions, let me know, or feel free to take charge.

File: docs/meetups/index.rst [hunk 3]
```diff
@@ -56,6 +56,43 @@ Current Meetups: Asia
...
+* Philadelphia
+* Raleigh / Durham
+
+.. meetup-listing::
+    :region: Quorum-US-East
+
+Current Meetups: Quorum North America West
+------------------------------------------
+
+Several meetups are members of the Quorum North America East group:
```
Comment (plaindocs) [inline docs/meetups/index.rst]:
```suggestion
Several meetups are members of the Quorum North America West group:
```

**Review by plaindocs — APPROVED** [review]
Thanks for your patience @mjang 

To get the organizer data we'd have to load the list of relevant meetups and iterate over them, which isn't entirely trivial. What might be an easier would be to generate an anchor for each meetup in the regional listing, and link to those in your manual bullet list.

*Something like* adding an anchor in https://github.com/writethedocs/www/blob/3efa16177601082f5e07e0443a695129b92d8b72/docs/_templates/include/meetups/listing.jinja#L5 using the `meetup` key.

```
<a name="{{meetup['meetup']}} />
```
might work.

Otherwise, it looks much better than not showing this info pending a better fix, so I'll merge it in!

**Comment by plaindocs** [comment]
@barbaricyawps What do you think about migrating all of https://github.com/write-the-docs-quorum/quorum-meetups into the meetup organizer guide in https://github.com/writethedocs/www/tree/master/docs/organizer-guide/meetups somewhere?

**Comment by barbaricyawps** [comment]
@plaindocs , sure I'd be open to moving our materials over there. I think I have sufficient permissions to open a PR against that and I'd be fine with migrating those over. I can open up that PR either this weekend or the next and tag you and Mike on it.

**Pull request merged**